interfaces/about.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Bootstrap -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT"
+      crossorigin="anonymous"
+    />
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <!-- /Fonts -->
+    <!-- Css -->
+    <link rel="stylesheet" href="../css/main.css" />
+    <!-- /CSS -->
+    <!-- FONT AWSOME -->
+    <script
+      src="https://kit.fontawesome.com/cd227d1718.js"
+      crossorigin="anonymous"
+    ></script>
+    <!-- /FONT AWSOME -->
+    <script src="../main.js" defer></script>
+    <title>DevBox Conference 2022</title>
+  </head>
+  <body>
+    <!-- THE HEADER SECTION -->
+    <header id="header">
+      <div class="container-fluid bg-dark">
+        <div class="container-md d-none d-md-block">
+          <ul class="toolbar d-flex justify-content-end gap-3 list-unstyled">
+            <li>
+              <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li><a class="link-light" href="#">English</a></li>
+            <li><a class="link-light" href="#">My Page</a></li>
+            <li><a class="link-light" href="#">Logout</a></li>
+          </ul>
+        </div>
+      </div>
+      <nav class="navbar navbar-expand-lg | nav">
+        <div class="container-md">
+          <a class="navbar-brand d-none d-md-block" href="../index.html"
+            ><img src="../assets/logo/Logo.svg" alt="DevBox logo"
+          /></a>
+          <button
+            class="navbar-toggler border-0"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon btn"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav gap-md-5 ms-auto mb-2 mb-lg-0 text-center">
+              <li class="nav-item">
+                <a class="nav-link current" aria-current="page" href="#"
+                  >About</a
+                >
```

Comment:
- [ ]  [OPTIONAL]  When the user is on the about page, there's no way to go back to the home page, especially on mobile devices. I highly suggest making sure this link navigates to the home page instead. 😄

Reply:
Hi @jasonfelice, Thanks for making it easy, I was thinking about this the whole day 😅, totally agree it's hard to navigate to the home page so am just gonna add a home page link.